portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,133 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <title>Samuel Sandoval</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="shortcut icon" href="images/SS_Initials.png" type="image/SS_initials"/>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+      integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" type="text/css" href="style.css" />
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <!-- navigation bar -->
+    <nav class="navbar navbar-expand-md">
+      <a class="navbar-brand" href="#">Samuel Sandoval</a>
+      <button
+        class="navbar-toggler navbar-dark"
+        type="button"
+        data-toggle="collapse"
+        data-target="#main-navigation"
+      >
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="main-navigation">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link" href="https://instagram.com/samueljsandoval/"><img class="icon1"src="images/instagram.png" border="0" /></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="https://linkedin.com/in/samuelsandoval1/"><img class="icon2" src="images/linkedin.png"border="0"></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="https://github.com/samuelsandoval1/"><img class="icon2" src="images/github.png"border="0"></a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+
+    <header class="page-header header container-fluid">
+      <div class="description">
+        <h1> Hello there,</h1>
+        <h1> I'm Sam Sandoval, and this is my site.</h1>
+        <a href="#about">Get to know me </a>
+      </div>
+    </header>
+
+
+	<div id= "about" class="container features">
+		<div class="row">
```

Comment:
supernit: tabs being used here instead of spaces, leading to odd indent. I think the style guide suggests 2 spaces for indentation for all css/html.

Reply:
Ok formatted doc to be 2 spaces